scenarios/tracking/02_mot_challenge.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<ul><li>Let's move this function into model.py to keep the notebook uncluttered</li><li>The variable name "eval_result" is not clear, maybe rename to something like "exp_path"?</li><li>Maybe rename accs to accumulators? We use acc in other places to refer to accuracy.</li><li>n_frame does not seem to be used</li><li>This would look nicer on two separate lines (black will change this anyway):</li></ul><pre class="ql-syntax" spellcheck="false">timer_avgs, timer_calls = [], []
</pre><ul><li>Can im_or_video_path be im or video, or for MOT it's always video? If so then rename.</li><li>[minor] rename result_file_path to result_path, the "file" seems unnecessary (and that would then be more in line with the rest of CVBP)</li><li>Can you add a one-line-comment how frame_rate is being set, ie the "10:meta_..."</li><li>Bad practice to have run_eval which if set to True ignores result_file_path, and bad_practice to ignore run_eval if the file is not present. Maybe just remove run_eval since you set it to False but I guess this is being ignored anyway, and then always execute the code block (ie no "if" anymore)</li><li>Is it important for the user to know where the file is or even that (under the hood) we use a results file? If not then could remove the print("Saved tracking results to...")</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
It's better user to know the tracking results, because those will be used for 'evaluate on test set' section. User can upload their tracking files to mot eval server.